### DISCUSSION TITLE
Go back to collection overview when saving a record

### DISCUSSION THREAD

**Discussion by tvanbeek** [payloadcms/payload#1190] [Feature Requests & Ideas]
After saving a new or edited record in a collection in the admin UI, a message appears saying the the record has been successfully saved. However, I would expect to return to the collection overview after clicking Save.

<img width="755" alt="image" src="https://user-images.githubusercontent.com/3518980/193251965-e6d15ba4-f975-43b1-ac59-c052c168a91f.png">

**Comment by bigmistqke** [comment]
This would be very unergonomic with big entries, where you might want to save while you are still in the progress of entering the data.
